includes/class-webmention-receiver.php

```diff
@@ -557,9 +484,11 @@ public static function webmention_verify( $data ) {
 			include_once ABSPATH . 'wp-includes/kses.php';
 		}
 
-		$remote_source = wp_kses_post( $remote_source_original );
-		$content_type  = wp_remote_retrieve_header( $response, 'Content-Type' );
-		$commentdata   = compact( 'remote_source', 'remote_source_original', 'content_type' );
+		$commentdata = array(
+			'content_type'           => $request->get_content_type(),
+			'remote_source_original' => $request->get_body(),
+			'remote_source'          => $request->sanitize_html( $request->get_body() ),
```

Comment:
`sanitize_html` is a static function. What about moving it to the `functions.php' as a global function?

Reply:
Would need a prefix, but no objection.